### PR TITLE
Helm fix

### PR DESCRIPTION
**Pull request by RawAlsaadi**

### CODE REVIEW

File: k8s/myeventsappchart/values.yaml [hunk 1]
```diff
@@ -32,23 +32,23 @@ databaseClusterIpService:
     targetPort: 5432
   type: ClusterIP
 dbConfigmap:
-  dbHost: database-cluster-ip-service
+  dbHost: myapp-database-cluster-ip-service
```
Comment (RawAlsaadi) [inline k8s/myeventsappchart/values.yaml:35]: @qimolin I had to hardcode "myapp", couldn't figure out how to do it another way. But this fixes the error:

>failed to connect to `host=database-cluster-ip-service ... no such host)

File: k8s/myeventsappchart/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: myeventsappchart
```
Comment (RawAlsaadi) [inline k8s/myeventsappchart/Chart.yaml:2]: @qimolin also changing this because of character limit errors we were getting

File: k8s/myeventsappchart/values.yaml [hunk 3]
```diff
@@ -32,23 +32,23 @@ databaseClusterIpService:
...
     image:
-      repository: https
-      tag: //ghcr.io/noleu/event_calendar_frontend
+      repository: ghcr.io/noleu/event_calendar_frontend
+      tag: latest
     resources:
       limits:
-        cpu: "2"
-        memory: 768Mi
+          cpu: 500m
```
Comment (RawAlsaadi) [inline k8s/myeventsappchart/values.yaml:47]: Changed this to make it work on my machine

File: .github/workflows/publish-ghcr.yaml [hunk 4]
```diff
@@ -13,8 +13,8 @@ on:
   push:
     branches: 
       - main
-      - feat/k8
-      - fix/docker-multiplatform-build
+      - feat/helm
```
Comment (qimolin) [inline .github/workflows/publish-ghcr.yaml]: Why did you add these branches here?

**Review by qimolin — APPROVED** [review]
